bitrepository-webclient/src/main/webapp/integrity-service.html

```diff
@@ -417,6 +558,56 @@ <h3 id="modalPagerLabel">Modal header</h3>
         $("#collectionChooser").val(collectionID).change();
     }
 
+    function getIntegrityStatus() {
```

Comment:
Duplicate method

Reply:
One more result of rebasing and resolving merge conflicts not going well. I think that it is fixed now, please check.